File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Struct/StructSamples.vb
Public Structure FooType
End Structure

Public Partial Structure ParentType
    Public Partial Structure NestedType
    End Structure
End Structure

Public Partial Structure ParentType
    Public Partial Structure AnotherNestedType
        Public Structure DeeperNestedType
        End Structure
    End Structure
End Structure

Namespace FooNamespace
    Public Structure InNamespaceType
    End Structure
End Namespace

Namespace FooNamespace.BarNamespace
    Public Partial Structure ParentType
        Public Structure NestedType
        End Structure
    End Structure
End Namespace

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Struct/StructSamples.cs
public struct FooType { }

public partial struct ParentType
{
    public partial struct NestedType { }
}

public partial struct ParentType
{
    public partial struct AnotherNestedType
    {
        public struct DeeperNestedType { }
    }
}

namespace FooNamespace
{
    public struct InNamespaceType { }
}

namespace FooNamespace.BarNamespace
{
    public partial struct ParentType
    {
        public struct NestedType { }
    }
}